src/components/TitleSubtitle/TitleSubtitle.css

```diff
@@ -0,0 +1,33 @@
+.section-title-black {
+    text-align: center;
+    display: block;
+    color: var(--color-text);
+    margin: 0 0 20px 0;
+    font-weight: bold;
+
+}
+
+h2 {
```

Comment:
In principe willen we wegblijven van type selectors _tenzij_ we hele algemene stylingregels aan het toepassen zijn. Algemene typografie regels zetten we in `index.css`, dus die mag je ook daar aanpassen en hier weghalen :) In componenten wil je daarom altijd gebruik maken van classes als je de algemene regels overschrijft.